pydicom/charset.py

```diff
@@ -51,18 +51,72 @@
     'GBK': 'GBK',  # from DICOM correction CP1234
 }
 
+# Map Python encodings to escape sequences as defined in PS3.3 in tables
+# C.12-3 (single-byte) and C.12-4 (multi-byte character sets).
+escape_codes = {
+    'ascii': b'\x1b(B',  # used to switch to ASCII G0 code element
+    'iso8859': b'\x1b-A',
+    'shift_jis': b'\x1b)I',
+    'iso2022_jp': b'\x1b$B',
+    'iso8859_2': b'\x1b-B',
+    'iso8859_3': b'\x1b-C',
+    'iso8859_4': b'\x1b-D',
+    'iso_ir_126': b'\x1b-F',
+    'iso_ir_127': b'\x1b-G',
+    'iso_ir_138': b'\x1b-H',
+    'iso_ir_144': b'\x1b-L',
+    'iso_ir_148': b'\x1b-M',
+    'iso_ir_166': b'\x1b-T',
+    'euc_kr': b'\x1b$)C',
+    'iso-2022-jp': b'\x1b$(D',
+    'iso_ir_58': b'\x1b$)A 58',
+}
+
 default_encoding = "iso8859"
 
+ESCAPE = b'\x1b'
```

Comment:
It took me like an hour to realise this was literally an 'ESCAPE' character rather than just a random byte string. Spend most of that time reading the standard to try and find where it came from...

Reply:
Sorry for that - I renamed it back to ESC as I had originally and added a comment...